400: {"message":"Failed to get video info: ERROR: could not find chrome cookies database in \"/home/runner/workspace/.config/google-chrome\"\nTraceback (most recent call last):\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/bin/.yt-dlp-wrapped\", line 9, in <module>\n sys.exit(main())\n ^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/__init__.py\", line 1072, in main\n _exit(*variadic(_real_main(argv)))\n ^^^^^^^^^^^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/__init__.py\", line 970, in _real_main\n with YoutubeDL(ydl_opts) as ydl:\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py\", line 973, in __exit__\n self.close()\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py\", line 976, in close\n self.save_cookies()\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py\", line 969, in save_cookies\n self.cookiejar.save()\n ^^^^^^^^^^^^^^\n File \"/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/functools.py\", line 1001, in __get__\n val = self.func(instance)\n ^^^^^^^^^^^^^^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py\", line 4093, in cookiejar\n return load_cookies(\n ^^^^^^^^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/cookies.py\", line 94, in load_cookies\n extract_cookies_from_browser(browser_name, profile, YDLLogger(ydl), keyring=keyring, container=container))\n ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/cookies.py\", line 115, in extract_cookies_from_browser\n return _extract_chrome_cookies(browser_name, profile, keyring, logger)\n ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n File \"/nix/store/mj7z8g8zfm3nd2ihymkk83czk9yz4xzd-python3.11-yt-dlp-2024.5.27/lib/python3.11/site-packages/yt_dlp/cookies.py\", line 294, in _extract_chrome_cookies\n raise FileNotFoundError(f'could not find {browser_name} cookies database in \"{search_root}\"')\nFileNotFoundError: could not find chrome cookies database in \"/home/runner/workspace/.config/google-chrome\"\n"}